Documentation/data-collection.md

```diff
@@ -21,318 +37,495 @@ data:
+    #
+    # Track current mode the cloud-credentials-operator is functioning under.
     - '{__name__="cco_credentials_mode"}'
     #
-    # (@openshift/storage, OpenShift Storage team) Persistent Volume usage metrics: this is the number of volumes per plugin
+    # (@openshift/storage, OpenShift Storage team)
+    #
+    # Persistent Volume usage metrics: this is the number of volumes per plugin
     # and per volume type (filesystem/block)
     - '{__name__="cluster:kube_persistentvolume_plugin_type_counts:sum"}'
     #
-    # (Advanced Cluster Management, @open-cluster-management/squad-kui-admins) visual_web_terminal_sessions_total is the count of Visual Web Terminal sessions created 
+    # (Advanced Cluster Management, @open-cluster-management/squad-kui-admins)
+    #
+    # visual_web_terminal_sessions_total is the count of Visual Web Terminal sessions created 
     # on the hub cluster.
     - '{__name__="visual_web_terminal_sessions_total"}'
     #
-    # (Advanced Cluster Management, @open-cluster-management/cluster-lifecycle-admin) acm_managed_cluster_info provides Subscription watch and other information for the managed clusters for an ACM Hub cluster.
+    # (Advanced Cluster Management, @open-cluster-management/cluster-lifecycle-admin)
+    #
+    # acm_managed_cluster_info provides Subscription watch and other information for the managed clusters for an ACM Hub cluster.
     - '{__name__="acm_managed_cluster_info"}'
     #
-    # (@openshift/storage, OpenShift Storage team) VMWare vCenter info: version of the vCenter where cluster runs
+    # (@openshift/storage, OpenShift Storage team)
+    #
+    # VMWare vCenter info: version of the vCenter where cluster runs
     - '{__name__="cluster:vsphere_vcenter_info:sum"}'
     #
-    # (@openshift/storage, OpenShift Storage team) The list of ESXi host versions used as host for OCP nodes.
+    # (@openshift/storage, OpenShift Storage team)
+    #
+    # The list of ESXi host versions used as host for OCP nodes.
     - '{__name__="cluster:vsphere_esxi_version_total:sum"}'
     #
-    # (@openshift/storage, OpenShift Storage team) The list of virtual machine HW versions used for OCP nodes.
+    # (@openshift/storage, OpenShift Storage team)
+    #
+    # The list of virtual machine HW versions used for OCP nodes.
     - '{__name__="cluster:vsphere_node_hw_version_total:sum"}'
     #
-    # (@openshift/team-build-api, OpenShift Build API Team) openshift:build_by_strategy:sum measures total number of builds on a cluster, aggregated by build strategy.
+    # (@openshift/team-build-api, OpenShift Build API Team)
+    #
+    # openshift:build_by_strategy:sum measures total number of builds on a cluster, aggregated by build strategy.
     - '{__name__="openshift:build_by_strategy:sum"}'
     #
-    # (@red-hat-data-services/odh, Open Data Hub team) This is (at a basic level) the availability of the RHODS system.
+    # (@red-hat-data-services/odh, Open Data Hub team)
```

Comment:
same comment regarding the org, people outside of `red-hat-data-services` will not be able to reach out to them

Reply:
I've chosen to link to https://github.com/red-hat-data-services/odh-deployer which is where the metric is declared.